docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,255 @@
+```
+
+### Create a worker
+
+1. In `src/main/java/io/camunda/demo/process_payments/`, create a file called `ChargeCreditCardWorker.java`.
+2. In the file created above, paste the following dependencies and package `package io.camunda.demo.process_payments`:
+
+```java
+package io.camunda.demo.process_payments;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import io.camunda.zeebe.spring.client.annotation.JobWorker;
+import io.camunda.zeebe.spring.client.annotation.Variable;
+```
+
+3. Next, we can decorate our class with `@Component` and instantiate a logger:
+
+```java
+@Component
+public class ChargeCreditCardWorker {
+
+  private final static Logger LOG = LoggerFactory.getLogger(ChargeCreditCardWorker.class);
+```
+
+4. Add the following method and decorate the `chargeCreditCard` method with `@JobWorker`:
+
+```java
+  @JobWorker(type = "charge-credit-card")
+```
+
+5. Specify a `@Variable(name = "totalWithTax") Double totalWithTax` argument into the `chargeCreditCard` method:
+
+```java
+  public Map<String, Double> chargeCreditCard(@Variable(name = "totalWithTax") Double totalWithTax) {
+```
+
+6. Lastly, implement the `chargeCreditCard` method:
+
+```java
+   LOG.info("charging credit card: {}", totalWithTax);
+    return Map.of("amountCharged", totalWithTax);
+```
+
+:::note
+If you want to check your work, visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code.
+:::
+
+In your terminal, run `mvn spring-boot:run`, where you should see the `charging credit card` output. In Operate, refresh if needed, and note the payment has executed.
+
+## Step 7: Start a process instance
+
+To kick off your process instance, take the following steps:
+
+1. In `ProcessPaymentsApplication.java`, instantiate a static logger:
+
+```java
+@SpringBootApplication
+
+private static final Logger LOG = LoggerFactory.getLogger(ProcessPaymentsApplication.class);
+```
+
+2. Declare `ZeebeClient` within the `@Autowired` annotation:
+
+```java
+	@Autowired
+	private ZeebeClient zeebeClient;
+
+	public static void main(String[] args) {
+		SpringApplication.run(ProcessPaymentsApplication.class, args);
+	}
+```
+
+3. To convert the application to a `CommandLineRunner`, add `implements CommandLineRunner` to the class declaration `public class ProcessPaymentsApplication implements CommandLineRunner`.
+4. Implement an overriding run method:
+
+```java
+	@Override
+	public void run(final String... args) {
+		var processDefinitionKey = "process-payments"; // or whatever the key is
+		var event = zeebeClient.newCreateInstanceCommand()
+				.bpmnProcessId(processDefinitionKey)
+				.latestVersion()
+				.variables(Map.of("total", 100))
+				.send()
+				.join();
+		LOG.info(String.format("started a process: %d", event.getProcessInstanceKey()));
+	}
```

Comment:
If we do add the `public class ProcessPaymentsApplication implements CommandLineRunner {` line earlier in the steps, we'll also need to account for another closing curly brace here.